src/ORM/Hierarchy/Hierarchy.php

```diff
@@ -24,7 +24,6 @@
  *
  * @property int $ParentID
  * @property DataObject|Hierarchy $owner
- * @method DataObject Parent()
```

Comment:
This one needs to stay - see the (non-standard) `get_extra_config()` method which is adding the `has_one` relation.